Backend/controllers/CaseInvestigationController.js

```diff
@@ -41,12 +65,13 @@ exports.getCaseInvestigations = async (req, res) => {
 
 // Fetch a case investigation by id
 exports.getCaseInvestigationById = async (req, res) => {
+    var { CaseInvestigation, Name_, Name } = demo_or_real_db(req.query);
     const { id } = req.params;
     try {
         const investigations = await CaseInvestigation.find({ _id: id });
         const result = await CaseInvestigation.populate(investigations, [
-            { model: 'Patient', path: 'patient_id', select: '_id first_name last_name' },
-            { model: 'User', path: 'assigned_to', select: '_id username' }
+            { model: 'Patient' + Name, path: 'patient_id', select: '_id first_name last_name' }, // Name added in case of demo db
```

Comment:
Keep in mind that changing the Name attribute to reflect a change of spaces would have impact on other models like Patient here. Maybe separating Name into several attributes will perhaps help solve the issue